serving/docs/lmi/lmi_input_output_schema.md

```diff
@@ -68,8 +70,7 @@ Apart from these common parameters, there are other parameters that are specific
 
 ```
 DeepSpeedRollingBatchParameters : {
-    'typical_p' : float (default= 1.0),
-    'do_sample' : boolean (default = False), 
+    'typical_p' : float (default= 1.0), 
     'seed' : integer (default = 0),
```

Comment:
should we remove seed from here since it is marked as a common parameter?

Reply:
Done